webapp/src/editortoolbar.tsx

```diff
@@ -208,6 +209,39 @@ export class EditorToolbar extends data.Component<ISettingsProps, EditorToolbarS
         this.props.parent.pairAsync();
     }
 
+    protected onCannotPairClick = async () => {
+        pxt.tickEvent("editortools.pairunsupported", undefined, { interactiveConsent: true });
+        const reasonUnsupported = await pxt.usb.getReasonUnavailable();
+        let modalBody: string;
+        switch (reasonUnsupported) {
+            case "security":
+                modalBody = lf("WebUSB appears to have been disabled by browser policy; check your browser's policy page at chrome://policy for more info.");
+                break;
+            case "oldwindows":
+                modalBody = lf("WebUSB is not available on Windows devices with versions below 8.1.");
```

Comment:
It looks like the version number above was listed as 6.3. (Line 573 of webusb.ts)

Reply:
The version listed in the userAgent is misleading, see the comment above it -- Windows NT 6.3 maps to windows8.1 consumer (it looks like with 10+ they made it just map how you would think it does)